.travis.yml

```diff
@@ -1,48 +1,78 @@
-os: osx
+language: generic
 # note: xcode9.4/macOS 10.13 is the default for macOS
```

Comment:
This comment should go above line 21 now that you are using the build matrix feature.

Reply:
Fixed :+1: